tests/testthat/test-mpmin-mpmax.R

```diff
@@ -0,0 +1,92 @@
+testMagpie <- function(regions = "AFR.1", years = "2000", names = "test", values = c(123), ...) {
```

Comment:
Huh, we usually use integers (actually numeric) for years, or strings but prefixed with a y, but apparently string without y also works :shrug: 
```suggestion
testMagpie <- function(regions = "AFR.1", years = "y2000", names = "test", values = c(123), ...) {
```